docs/FAQ.rst

```diff
@@ -31,4 +32,37 @@ This will be because you have a version of setuptools that is too old, Gilbert r
 
 You can fix this by updating versions:
 
-    pip install -U setuptools
\ No newline at end of file
+.. code-block:: sh
+
+    pip install -U setuptools
+	
+How do I have Gilbert automatically rebuild the site when files change?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
My personal preference is to have the header strings match the length of the header.